backend/src/middleware/handleHtmlContent/hashtags/extractHashtags.js

```diff
@@ -0,0 +1,22 @@
+import cheerio from 'cheerio'
+const ID_REGEX = /\/search\/hashtag\/([\w\-.!~*'"(),]+)/g
+
+export default function(content) {
```

Comment:
Did you test `https://www.example.org/#anchor` for false positives? I am pretty sure the used method does handle that correctly - just to make sure we dont run into problems here

Reply:
Good idea. @ulfgebhardt 
I will do tomorrow …

Reply:
Done @ulfgebhardt ❤️